pylint_sqlalchemy/orm.py

```diff
@@ -0,0 +1,44 @@
+"""Pylint sqlachemy hack module."""
+
+
+# pylint disable=R0205
```

Comment:
You are missing a `:`, the format is `# pylint: disable=R0505` or, preferably `# pylint: disable=useless-object-inheritance` and it's typically best to disable inline with the code that causes the problem so that we don't have to enable later.

In this case, however, I have added this to the project's `.pylintrc` ignore list since it's a Python 2/3 compatibility issue and we still want to support 2.7.

You should, hopefully, be able to remove this line and rebase on master to resolve this issue.

Reply:
ok, that would be great